### PR TITLE
Pull current feedback and link optimisation into dev branch

### PR DESCRIPTION
**Pull request by huachen24**
Merging into dev so that future works are not too outdated
rqt button for resume syncthing has some errors
rqt go/return button sends commands to aircraft 3 no matter the selection
check rssi currently removed

### CODE REVIEW

File: ogc_ws/src/telegram/src/telegram_link [hunk 1]
```diff
@@ -175,18 +208,19 @@ if __name__ == '__main__':
...
+		pub_to_timeout = rospy.Publisher('ogc/to_timeout', LinkMessage, queue_size = 5)
 		# inform all valid nodes that the link is up
 		# td.send_message_multi(valid_ids, ("Air" if is_air else "Ground") + " id " + str(self_id) + " Setup complete")
 
 		rospy.spin()
 
 	except rospy.ROSInterruptException:
 		sys.stdout.flush()
-		td.destroy()
+		# td.destroy()
```
Comment (huachen24) [inline ogc_ws/src/telegram/src/telegram_link]: should this be commented?

File: ogc_ws/src/despatcher/scripts/timeoutscript.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            message.data = "single tick"
+        elif state == 2:
+            message.data = "double tick"
+    return message
+
+def increment():
+    global uuid
+    uuid += 1
+    if uuid > 255:
+        uuid -= 255
```
Comment (rumesh986) [inline ogc_ws/src/despatcher/scripts/timeoutscript.py:25]: Could this be uuid = 0?

File: ogc_ws/src/despatcher/src/timeout.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+def send_to_rqt(message_id, data):
+    message = LinkMessage()
+    message.id = message_id
+    message.data = data
+    message.uuid = 0 # 0 is for uuid for non-A2G message
+    pub_to_rqt.publish(message)
+
+if __name__=='__main__':
+    rospy.logerr("name is called")
```
Comment (rumesh986) [inline ogc_ws/src/despatcher/src/timeout.py:104]: Is this needed? or maybe change the message itself

File: ogc_ws/src/telegram/src/telegram_link (hunk 1 above)
Comment (rumesh986) [reply ogc_ws/src/telegram/src/telegram_link]: re enabled the destroy function

**Review by rumesh986 — APPROVED** [review]

**Review by danipurwadi — APPROVED** [review]

**Pull request merged**